po/fr.po

```diff
@@ -17,7 +17,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: util-linux 2.36-rc2\n"
 "Report-Msgid-Bugs-To: util-linux@vger.kernel.org\n"
-"POT-Creation-Date: 2020-07-09 12:54+0200\n"
+"POT-Creation-Date: 2020-07-22 12:05+0200\n"
 "PO-Revision-Date: 2020-07-23 08:09+0200\n"
```

Comment:
Please, don't add generated stuff to your patches, see
https://github.com/karelzak/util-linux/blob/master/Documentation/howto-contribute.txt#L31

It will be updated automatically before release.

(hmm....  it seems I need to add a note about po/ directory to the howto-contribute.txt)